modules/aerodyn/src/AeroDyn_AllBldNdOuts_IO.f90

```diff
@@ -766,7 +765,7 @@ SUBROUTINE Calc_WriteAllBldNdOutput( p, p_AD, u, m, m_AD, x, y, OtherState, Indx
          CASE ( BldNd_Mm )
             DO IdxBlade=1,p%BldNd_BladesOut
                DO IdxNode=1,p%NumBlNds   
-                  y%WriteOutput( OutIdx )  = m%M(IdxNode,IdxBlade)
+                  y%WriteOutput( OutIdx )  = m%Mz(IdxNode,IdxBlade)
```

Comment:
@ebranlard , I think I had an error in the original code I sent you where `m%M` wasn't always set, but it looks like you fixed that in `SetOutputsFromBEMT`. I think this should still be m%M.